api/src/main/java/org/openmrs/module/fhir2/api/dao/impl/FhirPersonDaoImpl.java

```diff
@@ -88,11 +88,5 @@ protected String getSqlAlias() {
 	protected void handleVoidable(Criteria criteria) {
 		criteria.add(eq("personVoided", false));
 	}
-	
-	@Override
-	protected Person voidObject(Person object) {
```

Comment:
It had a special handler because we had to call a special method (`setPersonVoided()`) because a Person record might also be a Patient (so voiding the Patient as a Person would void the Patient but not the Person)... The default handlers will already handle that logic more correctly than here though.